.github/release-please/.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-    ".": "1.16.10"
+    ".": "1.16.9"
```

Comment:
Will this cause issues when try to release? (I'm assuming we'll have to release as 1.16.11)

Reply:
Probably it would fail, in a following [commit](https://github.com/grafana/synthetic-monitoring-app/pull/1027/commits/47e1a597cb7fd52448e0548ae75e186d3d9b0d56) I changed the version back to `1.16.10` as it was reverted automatically when running `git revert v1.16.9..HEAD`